build/sg3-utils/patches/solaris.patch

```diff
@@ -0,0 +1,50 @@
+ 
```

Comment:
Can you add a line saying where you got this patch from please?
It looks like it originally came from solaris-userland, if so just leave a link back to the source.